package.json

```diff
@@ -23,14 +23,14 @@
     "underscore": "1.7.0"
   },
   "devDependencies": {
-    "eslint": "^2.7.0",
+    "eslint": "^4.19.1",
```

Comment:
Any breaking changes with this upgrade

Reply:
Dev dependency it will not affect at all. It is tested checked in the CI. It is used to check if the file is format properly.